Makefile

```diff
@@ -0,0 +1,32 @@
+SHELL := /bin/bash
+
+CONFIG_PATH=config.toml
+
+deploy-contract:
+	@chmod +x ./contracts/deploy.sh
+	@./contracts/deploy.sh
+	@cp contracts/out/ZkAuction.sol/ZkAuction.json assets/ZkAuction.json
+
+test-auction:
+	RUST_BACKTRACE=1 cargo test --release --color=always --message-format=json-diagnostic-rendered-ansi --no-run --package zk_auction --lib tests::test_auction_service::test_auction_service --profile dev
```

Comment:
Don't copy-paste RustOver command to here 
```suggestion
	cargo test --package zk_auction --lib tests::test_auction_service::test_auction_service --profile dev
```
--no-run ??
<img width="302" alt="Screenshot 2024-10-30 at 11 30 19" src="https://github.com/user-attachments/assets/6508818e-742a-4aa3-b579-386e1f82bf62">